sklearn/ensemble/_bagging.py

```diff
@@ -257,12 +262,13 @@ class BaseBagging(BaseEnsemble, metaclass=ABCMeta):
         "n_jobs": [None, Integral],
         "random_state": ["random_state"],
         "verbose": ["verbose"],
+        "base_estimator": [HasMethods(["fit", "predict"]), StrOptions({"deprecated"})],
```

Comment:
I guess None should still be a valid option for base_estimator. Otherwise we might break user code if they explicitely set ``base_estimator=None``.